responses/de/HassTurnOn.yaml

```
@@ -8,6 +8,6 @@ responses:
       cover: "{{ slots.name }} geöffnet"
       cover_area: "{{ slots.area }} geöffnet"
       open_all: "alle geöffnet"
-      lock: "{{ slots.name }} abgeschlossen"
+      lock: "abgeschlossen"
```

we should have different responses for targeting a single entity or all
```suggestion
      lock: "{{ slots.name }} abgeschlossen"
      lock_all: "alle abgeschlossen"
```

---

hi @steffenrapp please address this comment in a new PR, thx 👍

---

Hey @mib1185 do you have a good recommendation? According to the [response style guide](https://developers.home-assistant.io/docs/voice/intent-recognition/style-guide) names of the entities or area should not be repeated so I don't have a good idea right now 🤔

---

we should at least differ between "abgeschlossen" and "alle abgeschlossen". same counts for the other responses

---

Have you seen dontinellis review about that topic? It makes a lot of sense to me. It can be more confusing than beneficial to have separate answers in this case. 😕 
Furthermore in English there's also no difference between single and all – probably due to the same reason ...

---

do you have a reference to this review?

---

It's inside of this PR. Don't know how to share a link.

---

ah, found it ... ok, let's orient on the English approach - maybe I wanted to be to "German precise" in this case 🙈

---

Haha alright 😄